### PR TITLE
Feature/icky 323: Category Search Implementation

### PR DESCRIPTION
**Pull request by kibo-chandradeeptalaha**

### CODE REVIEW

File: components/KiboPagination.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="pagination">
+  <div v-if="pagination.totalItems > 20" class="pagination">
```
Comment (kibo-kevinwatts) [inline components/KiboPagination.vue]: Lets remove this v-if statement

File: pages/category/index.vue [hunk 2]
```diff
@@ -220,96 +190,99 @@
...
-    }
-
-    const selectFilter = (filterValue) => {
-      const qs = route.value?.query as { filters: string }
-      const filters = qs.filters?.split(",") || []
-      const currentIndex = filters.indexOf(filterValue)
-      if (currentIndex > -1) {
-        filters.splice(currentIndex, 1)
+    const categoriesFromSearch = computed(() => {
+      let header: string, children: FacetValue[]
```
Comment (kibo-kevinwatts) [inline pages/category/index.vue]: move this function to productSearchGetter, getCategoryFacet that returns the { header, children }

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]
This looks good, two small changes and resolve conflicts please

File: components/KiboPagination.vue (hunk 1 above)
Comment (kibo-chandradeeptalaha) [reply components/KiboPagination.vue]: Done

File: pages/category/index.vue (hunk 2 above)
Comment (kibo-chandradeeptalaha) [reply pages/category/index.vue]: Done

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**